main.go

```diff
@@ -58,6 +60,9 @@ func main() {
 		klog.Fatalf("Can not get cluster config. error: %v", err)
 	}
 
+	cfg.QPS = float32(*qps)
```

Comment:
Thanks @benjaminkz for the fix. Would it be possible to add a test that adds `qps` and `burst` as configurable parameters ?

Reply:
I printed the value of them to the console, where I can see in eks event logging. They were well received by the provider.